rp_transferto/tests/test_tasks.py

```diff
@@ -106,3 +106,107 @@ def test_successsful_run_with_cached_msisdn(
         self.assertTrue(fake_topup_data.called)
         self.assertTrue(fake_get_contacts.called)
         self.assertTrue(fake_update_contact.called)
+
+
+@override_settings(
```

Comment:
If these never change we could always move them to the test settings file. Nitpicking.

Reply:
Nah, not nitpicking - that's a good point, I'll do that now :) thanks